.github/test-a-feature.md

```diff
@@ -103,4 +109,49 @@ yarn test
\ No newline at end of file
+[1]: https://github.com/stardust-ui/react/tree/master/test/specs/commonTests
+
+## Screener Tests
+
+For some components, it is necessary to write screenshot tests in order to check that they render properly. For each component example added to the docsite a screenshot test is automatically created. This checks if that the component is rendered in a consistent way, as it looks for visual differences between the previous rendering and the current one. We use [screener-io](https://screener.io/) to achieve screenshot testing.
+
+### Tests with Steps API
+
+This default test only checks the rendering for the component in its initial state. In order to test the rendering of more complex components, such as a `Dropdown`, screener provides an [api](https://www.npmjs.com/package/screener-runner) to execute actions on the DOM, in a way similar to end-to-end tests. These tests are executed by Screener as long as both the tests and their files respect the conventions:
+- the test file should be placed at the same location as the component example under test.
+- the test file should be named exactly as the component example file. If `DropdownExample.shorthand.tsx` is to be tested, the screener test file should be named `DropdownExample.shorthand.steps.ts`.
+- the tests should be written as an array of callbacks that accept a `steps` parameter, as all of them will be chained in `screener.config`. The `steps` parameter is actually the `Steps` object from screener, instantiated in `screener.config`.
+
+#### Example for a test file:
+
+```tsx
+import { Dropdown } from '@stardust-ui/react'
+
+const steps = [
+  steps => steps.click(`.${Dropdown.slotClassNames.triggerButton}`)
+    .snapshot('Opens dropdown list'),
+  steps => steps.hover(`.${Dropdown.slotClassNames.itemsList} li:nth-child(2)`)
+    .snapshot('Highlights an item'),
+]
+
+export default steps
+```
+
+#### Important mentions:
+
+- by convention, each test is written as a different callback and added to the steps array.
+- an actual assertion is performed by taking a `.snapshot(<Your test name here>)`, as the assertion is performed by comparing the screenshot with the one taken on the previous run and considered correct.
+- a test can have as many snapshots as needed.
+- before the snapshot is taken, steps are added to reach the state of assertion, with methods from `screener-runner` API (`.click(<selector>)`, `.hover(<selector>)`etc.)
+- tests do not perform any cleanup by default. This means a test is dependent on the state of the component in which the previous test leaves it. In the example above, the second test will receive the `Dropdown` component with the list open.
+
+### Run Screener tests
+In order to run the tests locally, make sure to have a Screener API key saved in the environment variables on your machine. For instance, on MacOS/Linux you can use `export SCREENER_API_KEY=<Your screener key here>`.
```

Comment:
And what is for Windows? 🔍 

This should work:
```bash
cross-env SCREENER_API_KEY=<Your screener key here> yarn test:visual
```

Reply:
oups. will address this in my arrow navigation PR :)